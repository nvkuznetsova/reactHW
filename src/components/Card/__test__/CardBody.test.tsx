import React from 'react';
import { shallow } from 'enzyme';

import { CardBody } from '..';

describe('Card', () => {
  const children = <div>Card Text</div>
  const component = shallow(<CardBody>{children}</CardBody>);

  it('should match snapshot', () => {
    expect(component).toMatchSnapshot()
  });
});