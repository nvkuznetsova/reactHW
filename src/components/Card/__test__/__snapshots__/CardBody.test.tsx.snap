// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Card should match snapshot 1`] = `
<div
  className="card-body"
>
  <div>
    Card Text
  </div>
</div>
`;
